test/quantization/test_quant_api.py

```diff
@@ -186,9 +186,14 @@ def test_8da4w_quantizer(self):
         assert isinstance(m.linear2, Int8DynActInt4WeightLinear)
         m(*example_inputs)
 
+    # TODO: save model weights as artifacts and re-enable in CI
```

Comment:
Until this is running in CI mind if we have an `eval.py` we can run in a new `scripts/` folder?

Reply:
Ok, will do in the next PR. Currently the models still live under test/ so we'll probably have to move those out as well